### PR TITLE
Fix typos in welcome/run-d-program-locally

### PR DESCRIPTION
**Pull request by wilzbach**
I put the link to the online docs upfront, because that's easier to reach if you are already in your browser. 

As `dmd`, `rdmd` and `dub` feature a similar text, which variant do you like most:

1a) Browse the [online documentation] or run `dub help` for an overview of available flags
1b) Browse the [online documentation] for an overview of available flags or run `dub help`.
2a) For an overview of available flags browse the [online documentation] or run `dub help`
2b) For an overview of available flags _either_ browse the [online documentation] or run `dub help`

(or suggest your own)

### CODE REVIEW

**Comment by stonemaster** [comment]
I vote for 1a. It's clear and concise

File: public/content/en/welcome/run-d-program-locally.md [hunk 1]
```diff
@@ -5,40 +5,40 @@ a package manager `dub`.
...
-Given a D file `test.d` a binary is created using *DMD* with this command:
+The *DMD* compiler compiles D file(s) and creates a binary.
+On the command line *DMD* can be invoked with the filename:
 
-    dmd test.d
+    dmd hello.d
 
-Run `dmd --help` or browse the [online documentation](https://dlang.org/dmd-linux.html#switches)
-for an overview of available flags.
+There are many options to modify the behavior of the *DMD* compiler.
```
Comment (wilzbach) [inline public/content/en/welcome/run-d-program-locally.md]: should "that allow" be added here?

Comment (stonemaster) [inline public/content/en/welcome/run-d-program-locally.md]: Yes, I think it soumds better that way.

**Comment by wilzbach** [comment]
> should "that allow" be added here?

Done ;-)

**Pull request merged**